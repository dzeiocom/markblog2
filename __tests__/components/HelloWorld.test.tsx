import React from 'react'
import renderer from 'react-test-renderer'
import HelloWorld from '../../src/client/components/HelloWorld'

/**
 * Example Test
 *
 * on first run it will generate a .snap file and after will compare result with it
 */

it('render a h1', () => {
	const tree = renderer.create((<HelloWorld>Test</HelloWorld>)).toJSON()
	expect(tree).toMatchSnapshot()
})
