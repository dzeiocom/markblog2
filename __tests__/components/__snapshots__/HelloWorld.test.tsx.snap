// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render a h1 1`] = `
<h1
  className="jsx-3954099876"
>
  Test
</h1>
`;
